index.bs

```diff
@@ -2222,6 +2259,84 @@ To <dfn>parse trigger specs</dfn> given a [=map=] |map|, a [=moment=]
 Issue: Invoke [=parse summary buckets=] and [=parse summary window operator=]
 from this algorithm.
 
+To <dfn>parse max number of event states</dfn> from a [=map=] map,
+a [=source type=] |sourceType|,
+a possibly null 32-bit unsigned integer |attributionScopeLimit|,
+and a [=randomized response output configuration=] |config|:
+1. If |attributionScopeLimit| is null:
+    1. If |map|["`max_num_event_states`"] [=map/exists=], return an error.
```

Comment:
Can we call this field `max_event_states` to match the format of `max_event_level_reports`? We don't use `num` in our other field names.

Reply:
Good point. Done.